frontend/pages/[resourceType].tsx

```diff
@@ -67,8 +49,24 @@ export default function ResourceList({ ids, resourceType }: InferGetServerSidePr
             marginRight: '150px'
           }}
         >
-          {idItems.length > 0 ? ( //if items exist
-            <ul>{idItems}</ul>
+          {resourceInfo.length > 0 ? ( // if items exist
+            <div>
+              <Stack style={{ marginTop: '10px', marginBottom: '10px', marginLeft: '50px', marginRight: '50px' }}>
+                {resourceInfo.map(res => {
+                  return (
+                    res && (
```

Comment:
Same here re: why we need a conditional render, this should always be defined?

Reply:
Good catch - originally the resource info array was constructed using a `.map` instead of a `.reduce` but I forgot to remove the conditional render after changing that logic